TombEngine/Objects/TR4/Object/tr4_sas_drag_bloke.cpp

```diff
@@ -0,0 +1,80 @@
+#include "framework.h"
+#include "tr4_sas_drag_bloke.h"
+#include "Specific/level.h"
+#include "Specific/Input/Input.h"
+#include "Game/Lara/lara.h"
+#include "Game/Lara/lara_helpers.h"
+#include "Game/items.h"
+#include "Game/pickup/pickup.h"
+#include "Specific/setup.h"
+#include "Game/health.h"
+#include "Math/Math.h"
+#include "Game/collision/collide_item.h"
+
+using namespace TEN::Input;
+using namespace TEN::Math;
+
+const auto DragSASPosition = Vector3i(0, 0, -460);
+const auto DragSasBounds = ObjectCollisionBounds 
+{
+	GameBoundingBox(
+		-BLOCK(1.0f / 4), BLOCK(1.0f / 4),
+		-100, 100, 
+		-BLOCK(1.0f / 2), -460
+	),
+		std::pair(
+			EulerAngles(ANGLE(-10.0f), ANGLE(-30.0f), 0),
+			EulerAngles(ANGLE(10.0f), ANGLE(30.0f), 0)
+		)
+};
+
+void DragSASCollision(short itemNumber, ItemInfo* laraItem, CollisionInfo* coll)
```

Comment:
Three-letter acronyms such as "SAS" may be treated like words and formatted as "Sas" everywhere.